.circleci/config.yml

```diff
@@ -389,12 +389,12 @@ workflows:
             - playmode-tests
 
       - publish-renderer:
-          filters:
-            branches:
-              only:
-                - master
-            tags:
-              only: /^\d+\.\d+\.\d+(.*)/ # npm tag + latest
+#          filters:
+#            branches:
+#              only:
+#                - master
+#            tags:
+#              only: /^\d+\.\d+\.\d+(.*)/ # npm tag + latest
```

Comment:
```suggestion
```

This may not be added next commit. The `if` in the `publish.ts` does the trick for us.